app/views/dashboards/show.html.erb

```diff
@@ -5,6 +5,11 @@
       <% @presenter.drafts.each do |draft| %>
         <div>
           <%= Dashboard::CollectionComponent.new(collection: draft.collection).name %> &gt; <%= link_to Works::DetailComponent.new(work: draft).title, edit_work_path(draft) %>
+          <% if allowed_to?(:delete?, draft) %>
```

Comment:
Could this section be it's own component and reused for views/dashboards/show.html.erb and in components/dashboard/collection_component.html.erb